plugin/trino-pulsar/src/main/java/io/trino/plugin/pulsar/PulsarColumnMetadata.java

```diff
@@ -0,0 +1,235 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.pulsar;
+
+import io.trino.spi.connector.ColumnMetadata;
+import io.trino.spi.type.Type;
+
+import java.util.Objects;
+
+public class PulsarColumnMetadata
+        extends ColumnMetadata
```

Comment:
Please use compostion instead of inheritance

Reply:
after closer look PulsarColumnMetadata is just carrying some additional properties, so I use properties of ColumnMetadata to carry such info when necessary and make PulsarColumnMetadata just a util class.